collectors/ebpf.plugin/ebpf_cgroup.h

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#ifndef NETDATA_EBPF_CGROUP_H
+#define NETDATA_EBPF_CGROUP_H 1
+
+#define NETDATA_EBPF_CGROUP_MAX_TRIES 3
+#define NETDATA_EBPF_CGROUP_NEXT_TRY_SEC 30
+
+#include "ebpf.h"
+#include "ebpf_apps.h"
+
+#define NETDATA_SERVICE_FAMILY "services"
+
+struct pid_on_target2 {
+    int32_t pid;
+    int updated;
+
+    netdata_publish_swap_t swap;
+
+    struct pid_on_target2 *next;
+};
+
+enum ebpf_cgroup_flags {
+    NETDATA_EBPF_CGROUP_HAS_PROCESS_CHART = 1,
+    NETDATA_EBPF_CGROUP_HAS_SWAP_CHART = 1<<2,
+    NETDATA_EBPF_CGROUP_HAS_SOCKET_CHART = 1<<3,
+    NETDATA_EBPF_CGROUP_HAS_FD_CHART = 1<<4,
+    NETDATA_EBPF_CGROUP_HAS_VFS_CHART = 1<<5,
+    NETDATA_EBPF_CGROUP_HAS_OOMKILL_CHART = 1<<6
```

Comment:
I guess this is for charts that have some apps integration right? So should we include the `shm` charts?

Reply:
I added it, but charts only with next PRs.